src/vmod_xkey.c

```diff
@@ -270,8 +282,9 @@ xkey_hashtree_insert(const unsigned char *digest, unsigned len)
 	if (key != NULL) {
 		xkey_hashhead_delete(&head);
 		CAST_OBJ_NOTNULL(head, (void *)key, XKEY_HASHHEAD_MAGIC);
-	} else
+	} else {
```

Comment:
no braces for single line control structures

Reply:
Thanks for your quick response! We'll implement these suggestions.

Reply:
I was around :-)